packages/salesforcedx-vscode-core/package.json

```diff
@@ -989,6 +989,11 @@
           "type": "boolean",
           "default": false,
           "description": "%enable_sobject_refresh_on_startup_description%"
+        },
+        "salesforcedx-vscode-core.activateDeployAndRetrieveForSourceTrackedOrgs": {
+          "type": "boolean",
+          "default": false,
+          "description": "%activate-deploy-and-retrieve-for-source-tracked-orgs%"
```

Comment:
@klewis-sfdc I think there are some more boilerplate changes that need to be made.  If you search the project for `enable-sobject-refresh-on-startup` you'll see what needs to be implemented, and activateDeployAndRetrieveForSourceTrackedOrgs should follow suit.

packages/salesforcedx-vscode-core/src/constants.ts should have something like `ACTIVATE_DEPLOY_AND_RETRIEVE_FOR_SOURCE_TRACKED_ORGS`

packages/salesforcedx-vscode-core/package.nls.json should be updated

packages/salesforcedx-vscode-core/src/index.ts should be updated

Reply:
nit: I think naming it _enable_ is better than _activate_ (eg `enableDeployAndRetrieveForSourceTrackedOrgs`) , since there is a president for "enable"  and we see it elsewhere in the code.

Reply:
Looks like there are also getters for the settings in the core package in sfdxCoreSettings.ts.